adoorback/notification/models.py

```diff
@@ -94,24 +127,38 @@ def create_or_update_notification(self, actor, user, origin, target, noti_type,
                                                                        content_en,
                                                                        content_ko,
                                                                        emoji)
+                noti_to_update.message_ko = updated_message_ko
+                noti_to_update.message_en = updated_message_en
 
-            noti_to_update.message_ko = updated_message_ko
-            noti_to_update.message_en = updated_message_en
             noti_to_update.is_visible = True
             noti_to_update.is_read = False
             NotificationActor.objects.create(user=actor, notification=noti_to_update)
             noti_to_update.notification_updated_at = timezone.now()
             noti_to_update.save()
             print(noti_to_update.message_ko)
             print(noti_to_update.message_en)
+            return noti_to_update
         else:
-            message_ko, message_en = construct_message(noti_type, actor.username + "님", None,
+            if target._meta.model_name == 'ping':
+                message_ko = f"{actor.username}님이 핑을 보냈습니다"
+                message_en = f"{actor.username} sent you a Ping!"
```

Comment:
We renamed ping to message! 
This should be changed to 
```suggestion
                message_ko = f"{actor.username}님이 메시지를 보냈습니다!"
                message_en = f"{actor.username} sent you a message!"
```